src/commonComponents/cart/components/cartButton/CartButton.js

```diff
@@ -0,0 +1,46 @@
+// Prop-Types
+import PropTypes from "prop-types";
+
+import _noop from "lodash/noop";
+
+// components
+import Button, { TYPES } from "../../../button";
+import PlusMinusButton from "../../../plusMinusButton";
+
+const CartButton = (props) => {
+  const { id, quantity, onCartChange } = props;
+
+  const onAddToCart = () => {
+    onCartChange(id, 1);
+  };
+
+  const onRemovalFromCart = () => {
```

Comment:
you can rename this because you are not removing everytime sometimes you are decrementing the quantity